### DISCUSSION TITLE
Структура narrative

### DISCUSSION THREAD

**Discussion by shamilnn** [fhir-ru/core#49] [General]
Предлагаемая структура для текстового описания к каждому профайлу, аналогичная как это собирает IG Publisher.

== Header ==

- Описание ресурса с точки зрения клинициста.
- Список обязательных элементов с кратким описание почему они обязательные
- Примеры использования (use case). Например, для Device – любое оборудование с госпитале с инвентарным номером; вживляемое оборудование/имплантат; органы на чипе.

== Differential / Snapshot View ==

Сам профайл

== Footer ==

Описание каждого значимого элемента в отдельности (например, .identifier, .name). Примеры заполнения для разных случаев.

== Search Parameters ==

Обязательные типы запросов, параметров поиска, операций и прочее согласно _CapabilityStatement_ для FHIR Server.
Примеры запросов.

**Comment by Vrodvi** [comment]
@eugenekogan-spb 

По поводу https://fhir.ru/meetings.2024-03-12

Чтобы не плодить новых дискуссий, продолжаем обсуждение в данной дискуссии - она подходит тематически.

---

> 1. Предложить описание футера в профиле, которое можно взять за образец

Предлагаю взять за образец канадское базовое руководство по реализации - оно как раз использует FSH для генерации профилей. 

В качестве конкретного примера можно посмотреть описание "подвала/footer" для профиля мед.работника https://build.fhir.org/ig/HL7-Canada/ca-baseline/StructureDefinition-profile-practitioner.html 

(см. исходное наполнение в формате FSH в гит-реопзитории https://github.com/HL7-Canada/ca-baseline/blob/db747fd006a2e58e57e16b09da001676065c997f/input/pagecontent/StructureDefinition-profile-practitioner-notes.md?plain=1#L2)

Наполнение такого "подвала", как мы и обсуждали, содержит пояснения по технической реализации конкретных частей/атрибутов профиля:

![image](https://github.com/fhir-ru/core/assets/92928137/05fda752-f7f1-4a72-969b-04c3ba191a46)

 В ZenDoc аналогичное описание, которое можно было бы разместить в качестве "подвала" к профилю, можно найти при описании атрибутов пациента - см. https://fhir.ru/core.profile.Patient#:elements .

**Comment by Vrodvi** [comment]
@eugenekogan-spb 
В прошлый раз обсуждение процесса формирования заголовков и подвалов велось тут - https://github.com/fhir-ru/core/discussions/250#discussioncomment-7799357

Ссылка на описание инструкции по работе с инструментом SUSHI для технического формирования заголовкой и подвалов описано тут - https://github.com/fhir-ru/core/discussions/235#discussioncomment-7764147

**Reply by Vrodvi** [reply]
@eugenekogan-spb 
По вопросу ссылок между двумя отдельными руководствами по реализации, затронутому на встрече, я нашёл пример настройки связей в американских руководствах ([mCode](https://github.com/HL7/fhir-mCODE-ig) и [US core](https://github.com/HL7/US-Core)):
- в mCode [профиль пациента](https://hl7.org/fhir/us/mcode/StructureDefinition-mcode-cancer-patient.html) является специализированной версией [профиля пациента](https://hl7.org/fhir/us/core/STU5.0.1/StructureDefinition-us-core-patient.html) из US core

Вот как эта взаимосвязь выглядит в файлах конфигурации этих двух руководств:

![image](https://github.com/fhir-ru/core/assets/92928137/51e7633f-9ab0-4d00-a977-fc3e84020eca)

**Reply by Vrodvi** [reply]
Сюда ещё интересное наблюдение:
в описании профиля руководства mCODE (в формате FSH) ссылка на профиль из другого руководства (US Core) указана в виде канонического URL,
но если смотреть на собранный сайт руководства mCODE, то там эта ссылка уже становится фактическим url профиля на сайте руководства US Core

![image](https://github.com/fhir-ru/core/assets/92928137/6104e865-3a1d-4c61-b61a-46b830d74849)
